hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -82,20 +82,27 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
 
         final var fileStore = handleContext.writableStore(WritableFileStore.class);
         final var fileUpdate = handleContext.body().fileUpdateOrThrow();
-        final var maybeFile = fileStore.get(fileUpdate.fileIDOrElse(FileID.DEFAULT));
```

Comment:
This is used for the regular files, not upgrade file, so perhaps it should be retained?

Reply:
It was moved further down in the same method

Reply:
That moved version clamps causes incorrect behavior for normal files, this version is correct.

Reply:
fixed